src/avfoundation.cs

```diff
@@ -8407,7 +8449,8 @@ interface AVVideoCompositionCoreAnimationTool {
 		AVVideoCompositionCoreAnimationTool FromComposedVideoFrames (CALayer [] videoLayers, CALayer inAnimationlayer);
 	}
 
-	[TV (11,0), NoWatch, iOS (11,0)]
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
+	[TV (11,0), NoWatch, Mac (10,13), iOS (11,0)]
```

Comment:
Mac (10,13)? I do not doubt you, I do not trust apple headers. @spouliot should intro pick this?

Reply:
@mandel-macaque yes - but only if the old-macOS bots are running the tests ;-)

Reply:
They should be... I'll check